cluster-up/cluster/images.sh

```diff
@@ -5,6 +5,7 @@ set -e
 declare -A IMAGES
 IMAGES[gocli]="gocli@sha256:e48c7285ac9e4e61fe0f89f35ac5f9090497ea7c8165deeadb61e464c88d8afd"
 if [ -z $KUBEVIRTCI_PROVISION_CHECK ]; then
+    IMAGES[k8s-fedora-1.17.0]="k8s-fedora-1.17.0"
```

Comment:
Create a push-k8s-fedora-1.17.0 prow job and update this has with the output from that.